demonstrations/quantum_volume.py

```diff
@@ -385,7 +385,7 @@ def qv_circuit_layer(num_qubits):
 with qml.tape.QuantumTape() as tape:
     qml.layer(qv_circuit_layer, m, num_qubits=m)
 
-print(tape.expand(stop_at=lambda op: isinstance(op, qml.QubitUnitary)).draw(wire_order=dev_ideal.wires, show_all_wires=True))
+print(tape.expand(stop_at=lambda op: isinstance(op, qml.QubitUnitary)).draw(wire_order=dev_ideal.wires, show_all_wires=True, show_matrices=True))
```

Comment:
```suggestion
expanded_tape = tape.expand(stop_at=lambda op: isinstance(op, qml.QubitUnitary))
print(qml.drawer.tape_text(expanded_tape, wire_order=dev_ideal.wires, show_all_wires=True, show_matrices=True))
```

I personally think we should favour the use of `qml.drawer.tape_text` over `tape.draw`.  This also splits the process into two lines, which may be more readable.